860_A1_Q4.ipynb

```diff
@@ -151,7 +215,7 @@
    "source": [
     "- As per the above result, The model is powerful and explains 83% of the variability in the Y value. Given that model has both Height and Experience as variable\n",
     "- Height poorly explains Y. (We can infer this from the fact that the coefficient value of Hieght is very low compared to Experince)\n",
-    "\n",
+    "[Paul] low coef value doesn't mean explanatory power is low. Results of F and t tests matters more.\n",
```

Comment:
I added this comment intuitively..You are right its not exactly correct...However, my understanding of p-value and F and t statistics in regression is  low..I am trying to understand those concepts and will update later